docs/learn/learn-parachains.md

```diff
@@ -9,13 +9,13 @@ slug: ../learn-parachains
 
 import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
 
-<MessageBox message="Parachain Slot Auctions will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. For existing parachains, the remainder of the lease will automatically be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
+<MessageBox message="[Agile Coretime](./learn-agile-coretime.md) is activated on the network, and parachain slot auctions and crowdloans have been deprecated. For existing parachains, the remainder of the lease is automatically converted to coretime. See more information [here](./learn-agile-coretime.md#implementation). For decentralized, transparent, and regulatory-compliant fundraising within the ecosystem, check out the [Polimec parachain](https://www.polimec.org/)." />
 
 :::info Testing on Rococo
```

Comment:
Rococo will be deprecated on Oct 14th. Will add an issue on the Wiki to replace its mentions with Paseo

Reply:
Added, see https://github.com/w3f/polkadot-wiki/issues/6284